pyproject.toml

```diff
@@ -1,178 +1,29 @@
-# see documentation, e.g.
-# - https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#declaring-project-metadata
-# - https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html
-# - https://www.python.org/dev/peps/pep-0621/
-
-[build-system]
-requires = ["setuptools>=64.0.0", "setuptools-scm", "wheel"]
-build-backend = "setuptools.build_meta"
-
-[project]
-authors = [
-    { name = "Zargham Ahmad", email = "zargham.ahmad@recetox.muni.cz" }
-]
-classifiers = [
-    "Development Status :: 2 - Pre-Alpha",
-    "Intended Audience :: Developers",
-    "Intended Audience :: Science/Research",
-    "License :: OSI Approved :: MIT License",
-    "Natural Language :: English",
-    "Programming Language :: Python :: 3",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-]
-dependencies = []
-description = "Tools for internal use"
-keywords = [
-    "rcx",
-    "process metadata",
-]
-license = {file = "LICENSE"}
-name = "rcx_tk"
-readme = {file = "README.md", content-type = "text/markdown"}
-requires-python = ">=3.8"
+[tool.poetry]
+name = "rcx-tk"
 version = "0.1.0"
+description = "This package adjusts and cleans the metadata file provided by a user."
+authors = ["Your Name <you@example.com>"]
+license = "GPL-2"
```

Comment:
This can either point to the license file or should state that this is published under MT license.